using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-api-reference.md

```diff
@@ -0,0 +1,74 @@
+# Campaign API reference
+
+## Set linkage fields
+
+Allows Campaign to connect fields from separate databases and create a more developed and personalized messaging experience. Profile template-based messages that contain PII-based personalization are downloaded. The **linkage fields** are stored as a base64-encoded JSON string in memory and sent in a custom HTTP header 'X-InApp-Auth' in all future Campaign rules download requests until ACPCampaign::resetLinkageFields is invoked.
+The **linkage fields** are cleared cleared if the app is gracefully close, crashed, or privacy status is changed to opt out.
```

Comment:
Please remove the second instance of "cleared, change from "close" to "closed", and add a "the" before "privacy status."